pyanaconda/ui/gui/spokes/software.py

```diff
@@ -464,6 +461,19 @@ def _get_selected_addons(self):
 
         return retval
 
+    def _mark_addon_selection(self, grpid, active):
```

Comment:
Maybe `selected` instead of `active`? There is no gui in this function, so `active` does not make much sense.